community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/executionplan/CompiledExecutionResult.scala

```diff
@@ -37,7 +37,7 @@ import scala.collection.{Map, mutable}
  * Base class for compiled execution results, implements everything in InternalExecutionResult
  * except `javaColumns` and `accept` which should be implemented by the generated classes.
  */
-abstract class CompiledExecutionResult extends InternalExecutionResult {
+abstract class CompiledExecutionResult(completion: CompletionListener, statement:Statement=null) extends InternalExecutionResult {
```

Comment:
No, but it made sense from the point of view of `RewindableExecutionResult`. And to have the `null` contained within this class made it obvious from the point of view of `close()` to be able to handle `null`, so it serves a documentation purpose.